package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "start": "react-app-rewired start",
-    "start:debug": "node --inspect-brk node_modules/react-app-rewired/scripts/start.js",
+    "start:debugging": "node --inspect-brk node_modules/react-app-rewired/scripts/start.js",
```

Comment:
NO.

Reply:
Reason?

Reply:
`start:debug` and `build:debugging` is magic

Reply:
What an amazing rename! I am eager to learn more about the story behind..

Reply:
what's the point here?

Reply:
It's a neutral change. Why block on this? @Artoria2e5

Reply:
I am not blocking on this. I am just saying that changing this to a longer, harder-to-type thing without explaining any magic is arbitrary and should not be in this PR.